pymobiledevice3/services/afc.py

```diff
@@ -22,7 +22,7 @@
 from pygments import formatters, highlight, lexers
 from pygnuutils.cli.ls import ls as ls_cli
 from pygnuutils.ls import Ls, LsStub
-from tqdm import trange
+from tqdm.auto import trange
```

Comment:
what is the reason for this diff?

Reply:
In the case pymobiledevice3 is used outside CLI environment, ex. jupyter notebooks, tqdm may choose the best option possible. If you think it is not valuable, I revert the change, it is not a big thing.

Reply:
Nice :) Didn't know that